bin/build-script

```diff
@@ -10,10 +10,7 @@ const fetch = require('node-fetch');
 const fs = require('fs-extra');
 const isProduction = process.env.ELEVENTY_ENV;
 
-const inputFiles = [
-  'node_modules/dompurify/dist/purify.js',
```

Comment:
We don't use dompurify on the client side unless I overlooked something so let's remove it.

Reply:
EW pulls a feed via XHR.